src/main/java/pl/javastart/bootcamp/domain/admin/AdminUserController.java

```diff
@@ -29,5 +35,10 @@ public String user(Model model, @PathVariable Long id) {
         return "admin/user";
     }
 
+    @GetMapping("/admin/dodajUsunAdmin")
```

Comment:
To jest mało bezpieczne. Jeżeli ktoś wie, że przypisywanie roli odbywa się pod takim adresem, to może administratorowi wysłać emailem, czy nawet na facebooku adres typu abc.pl/admin/dodajUsunAdmin?id=123
gdzie 123 to identyfikator atakującego użytkownika. Taki link może być oczywiście ukryty za jakimś skracaczem.

W ten sposób bez żadnej weryfikacji przydzielimy jakiemuś randomowemu użytkownikowi rolę administratora. W przypadku żądania POST mamy w spring security gratis ochronę CSRF, więc coś takiego nie przejdzie. 
Zmień w szablonach zwykłe linki na formularze